ghost/core/core/server/data/migrations/versions/5.13/2022-09-02-12-55-rename-members-bio-to-expertise.js

```diff
@@ -0,0 +1,16 @@
+const logging = require('@tryghost/logging');
+const {createNonTransactionalMigration} = require('../../utils');
+
+module.exports = createNonTransactionalMigration(
+    async function up(knex) {
+        await knex.schema.table('members', (table) => {
+            table.renameColumn('bio', 'expertise');
```

Comment:
@daniellockyer Is this gonna work for SQLite? We bundle > 3.25 now right?

Reply:
We do - yes

I worry about performance though - we use `ALGORITHM=COPY` for adding and removing a column. I wonder if we should do the same here?

Reply:
Actually, in MySQL 8, the default is ALGORITHM=INSTANT, which is probably better